cinema/tests/test_movie_api.py

```diff
@@ -114,16 +112,12 @@
         }
 
         response = self.client.put(url, payload)
-        self.assertEqual(
-            response.status_code, status.HTTP_405_METHOD_NOT_ALLOWED
-        )
+        self.assertEqual(response.status_code, status.HTTP_405_METHOD_NOT_ALLOWED)
 
     def test_delete_movie(self):
         movie = sample_movie()
 
         url = detail_url(movie.id)
         response = self.client.delete(url)
 
-        self.assertEqual(
-            response.status_code, status.HTTP_405_METHOD_NOT_ALLOWED
-        )
+        self.assertEqual(response.status_code, status.HTTP_405_METHOD_NOT_ALLOWED)
```

Comment:
The `test_delete_movie` method expects a `405 Method Not Allowed` response. Verify that the API is set up to disallow DELETE requests on the movie detail endpoint. If DELETE requests are allowed, update the test to reflect the correct expected status code.

Reply:
agree, usually delete methor returns `302` status